src/Monocle/Backend/Index.hs

```diff
@@ -631,6 +649,17 @@ indexChanges changes = indexDocs $ fmap (toDoc . ensureType) changes
   toDoc change = (toJSON change, getChangeDocId change)
   ensureType change = change {echangeType = EChangeDoc}
 
+indexErrors :: MonoQuery :> es => IndexEffects es => [EError] -> Eff es ()
+indexErrors errors = indexDocs $ fmap toDoc errors
+ where
+  toDoc err = (getErrorDoc err, getErrorDocId err)
+
+  getErrorDoc :: EError -> Value
+  getErrorDoc err = object ["type" .= EErrorDoc, "error_data" .= toJSON err]
```

Comment:
It seems that error_data should be base64 encoded: https://www.elastic.co/guide/en/elasticsearch/reference/current/binary.html

Reply:
More info about the binary type https://medium.com/@sachin.frayne/elasticsearch-binary-data-type-9429f8b2db85